def.cf

```diff
@@ -258,6 +258,11 @@ bundle common def
         comment => "If reports are not collected for an extended period of time
                     the disk may fill up or cause additional collection
                     issues.";
+
+      "enable_inventory_ipv4_addresses" -> { "Mission Portal" }
```

Comment:
Don't see any reason to make this configurable, at least not on the top level.. it should be very performant.